ReactAndroid/src/main/jni/first-party/yogajni/Android.mk

```diff
@@ -10,7 +10,7 @@ LOCAL_SRC_FILES := \
 LOCAL_C_INCLUDES := $(LOCAL_PATH)/jni
 
 LOCAL_CFLAGS += -Wall -Werror -fvisibility=hidden -fexceptions -frtti -O3
-CXX11_FLAGS := -std=c++11
+CXX11_FLAGS := -std=c++14
```

Comment:
in clang lambdas you are using is part of c++14 extension.
the code continues to compile with your toolchain.